.circleci/config.yml

```diff
@@ -13,10 +13,12 @@ jobs:
       - run: git submodule update --remote --force
       - run: cd wp-e2e-tests && git checkout origin/${E2E_BRANCH-master}
       - restore_cache:
-          key: v1-{{ checksum "wp-e2e-tests/package-lock.json" }}
-      - run: cd wp-e2e-tests && npm install
+          keys:
+            - v1-{{ checksum "wp-e2e-tests/.nvmrc" }}-{{ checksum "wp-e2e-tests/package-lock.json" }}
+            - v1-{{ checksum "wp-e2e-tests/.nvmrc" }}
+      - run: cd wp-e2e-tests && npm ci
       - save_cache:
-          key: v1-{{ checksum "wp-e2e-tests/package-lock.json" }}
+          key: v1-{{ checksum "wp-e2e-tests/.nvmrc" }}-{{ checksum "wp-e2e-tests/package-lock.json" }}
           paths:
             - "wp-e2e-tests/node_modules"
```

Comment:
`npm ci` will completely remove `node_modules` and install fresh. Caching `node_modules` doesn't work well.

What we have been doing is cache in CI the npm cache. That would mean changing the paths to `~/.npm`. `node_modules` will be reinstalled, but `npm ci` can be smarter about pulling locally cached modules rather than hitting the network. We've seen good results.

[Here it is in Calypso](https://github.com/Automattic/wp-calypso/blob/3aa03213413e3ed0ddd422e9f6766668fe728b8a/.circleci/config.yml#L44-L48)

Reply:
Thanks, I will work on an update, does this mean wp-e2e-tests-canary needs an update also?

https://github.com/Automattic/wp-e2e-tests-canary/blob/master/.circleci/config.yml#L15

Reply:
[This section looks good](https://github.com/Automattic/wp-e2e-tests-canary/blob/718e8c689822384e099e1596f9ccc1a160d1152a/.circleci/config.yml#L15-L24), I don't believe any changes are required. `npm ci` is used and the npm cache (`~/.npm`) is what's being cached.

That said, any repos that have moved to `npm@>=5.7` should probably be using this general scheme. If you know of any that haven't been updated, let's do it 😀